### PR TITLE
[#159] Added Dex support to Airflow

### LINKED ISSUE #159: Secure Airflow with Dex (JWT)
Airflow should authorize user with JWT token passed in request header. This feature should be configurable.

### PR DESCRIPTION
**Pull request by aliaksandr-d**
Added Dex support to Apache Airflow.
It can be enabled and configured via profile.
This PR closes #159 but should be merged ONLY after PR #160

### CODE REVIEW

File: etl/setup.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                "Unable to find version string in %s." % (file_content,))
+
+
+setup(name='etl',
+      version=extract_version(
+          os.path.join(PACKAGE_ROOT_PATH, 'etl','version.py')),
+      description='External library for airflow',
+      url='http://github.com/legion/legion',
+      author='Legion team',
+      author_email='alexey@kharlamov.biz',
```
Comment (akharlamov) [inline etl/setup.py]: legion-dev@googlegroups.com

File: k8s/airflow/Dockerfile [hunk 2]
```diff
@@ -18,6 +18,9 @@ FROM ubuntu:16.04
 
 ENV AIRFLOW_HOME /home/airflow
 
+ARG pip_extra_index_params=""
+ARG pip_legion_version_string=""
+
 RUN apt-get update && apt-get install -y \
  wget \
  default-jdk \
```
[the remark this answers is not in the document]
Comment (aliaksandr-d) [reply k8s/airflow/Dockerfile]: Fixed.

**Comment by aliaksandr-d** [comment]
Fixed copyrights, codestyle. Renamed *etl* to *legion-airflow*

**Comment by aliaksandr-d** [comment]
Fixed. Waiting for review

**Comment by aliaksandr-d** [comment]
Test please

**Comment by aliaksandr-d** [comment]
Test this please

**Comment by aliaksandr-d** [comment]
Moved to separate PR, where all DEX functionality is aggregated.

**Pull request closed without merging**